File: python-2.5.2/comment.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# vi:encoding=utf-8

from TestEnv import TestEnv

te= TestEnv()

te.checkComplainAndAdjustExpected( 0)

# this is a comment

"""
 this is a comment
"""

te.checkComplainAndAdjustExpected( 0)

print te.result()

File: ruby/comment.rb
#!/usr/bin/ruby

load "TestEnv.rb"

te= TestEnv.new()

te.checkComplainAndAdjustExpected( 0)

# this is a comment


=begin
 this is a comment
=end

te.checkComplainAndAdjustExpected( 0)

print( te.result+ "\n")
